clients/node_client.go

```diff
@@ -117,9 +118,9 @@ func (c client) GetChunks(
 		return
 	}
 
-	chunks := make([]*core.Chunk, len(reply.GetChunks()))
+	chunks := make([]*encoding.Frame, len(reply.GetChunks()))
```

Comment:
since there are so many places where chunks are assigned to frames. Does it make sense to use encoding.Chunk? My only concerns against it is that chunk usually means data only. However, the data here contain both data and proof

Reply:
It might be a future cleanup action to make the variable names consistent.